src/main/cpp/blaze_util_linux.cc

```diff
@@ -161,8 +161,17 @@ string GetSystemJavabase() {
   return blaze_util::Dirname(blaze_util::Dirname(javac_dir));
 }
 
+// Relevant entries from /proc/[pid]/stat.
+// Run `man proc` and search "/proc/[pid]/stat"
+struct ProcessStatus {
+  // Process state. See "(3) state" in `man proc`.
+  string state;
+  // Start time since startup. See "(22) starttime" in `man proc`.
+  string start_time;
+};
+
 // Called from a signal handler!
-static bool GetStartTime(const string& pid, string* start_time) {
+static bool GetProcessStatus(const string& pid, ProcessStatus* status) {
```

Comment:
I wasn't sure whether it made more sense to have this just return `false` if the process if the zombie, instead of modifying it to return a struct. Perhaps someone more familiar with this code could comment.